platform/broadcom/sonic-platform-modules-dell/z9332f/sonic_platform/psu.py

```diff
@@ -119,13 +119,13 @@ def get_voltage_low_threshold(self):
         """
         Returns PSU low threshold in Volts
         """
-        return 11.4
+        return "11.4"
 
     def get_voltage_high_threshold(self):
         """
         Returns PSU high threshold in Volts
         """
-        return 12.6
+        return "12.6"
```

Comment:
According to the API definition in sonic-platform-base, these methods should return floats, not strings. What is the reason you feel the need to change to strings? Maybe there is a bug elsewhere?

Reply:
Hi Joe,
get_voltage() was returning string and high/low threshold returns float previously.
Because of this, comparison in psu daemon gets failed and fields are not updated in state DB.
Will change get_voltage and threshold API's to return float.